www/src/context/SettingsContext.jsx

```diff
@@ -46,11 +46,6 @@ const SettingsContextProvider = ({ children }) => {
             if you simply change href of the stylesheet there is a small window of time when the previous
             theme gets unapplied and new one loaded which leaves whose site without styles.
          */}
-        <link
-          href="/static/openedx-theme.css"
-          rel="stylesheet"
-          type="text/css"
-        />
```

Comment:
[inform] Rendering this here means the theme isn't present in `<head>` until after the first render of `SettingsContextProvider`, which causes a quick flash of unstyled content locally. This was instead moved to gatsby-ssr.js so the stylesheet gets added at build time instead.